webpack.config.js

```diff
@@ -140,6 +141,34 @@ module.exports = (env, argv) => {
           pageUrl: 'https://simpleicons.org',
         },
       }),
+      new HtmlWebpackPlugin({
+        inject: true,
+        filename: 'removed.html',
+        template: path.resolve(ROOT_DIR, 'removed.pug'),
+        templateParameters: {
+          removedIcons: removedIcons
+            .sort((v1, v2) => v1.title.localeCompare(v2.title))
+            .map((icon, iconIndex) => {
+              const luminance = getRelativeLuminance(`#${icon.hex}`);
+              return {
+                base64Svg: Buffer.from(icon.path).toString('base64'),
+                hex: icon.hex,
+                light: luminance < 0.4,
+                superLight: luminance > 0.95,
+                superDark: luminance < 0.02,
+                issue: icon.issue,
+                versionIssue: icon.versionIssue,
+                path: icon.path,
+                title: icon.title,
+              };
+            }),
+          iconCount: removedIcons.length,
+          pageTitle: 'Removed Icons',
+          pageDescription:
+            "Removed icons in all previous versions. Some icons can't be shown since they were removed because the brand does not allow them to be modified to fit this library.",
+          pageUrl: 'https://simpleicons.org/removed',
```

Comment:
A lot of configuration can also be reused between the two pages, this should help us maintaining the project.

Reply:
agreed.